lib/try.js

```diff
@@ -1,17 +1,75 @@
 var express = require('express');
 var request = require('request');
+var streamBuffers = require("stream-buffers");
 
 module.exports = function() {
   var proxy = new express.Router();
   
-  proxy.all('/', function(req, res) {
+  proxy.all('/', function( req, resp ) {
+    var requestedUrl = getRequestedUrl( req );
+    enrichResponseWithRequestHeaders( req, resp );
+    var requestBuffer = new streamBuffers.WritableStreamBuffer({
+      initialSize: (100 * 1024),      // start as 100 kilobytes.
+      incrementAmount: (10 * 1024)    // grow by 10 kilobytes each time buffer overflows.
```

Comment:
These amounts seem the defaults from StreamBuffer - is there sense in putting an implicit limit on the speed we can deal with requests? 
I'm all for back pressure, but this seems a little early to optimise for that sort of thing :-)